enterprise/management/commands/assign_enterprise_user_roles.py

```diff
@@ -0,0 +1,38 @@
+"""
+management command for assigning existing enterprise users to their respective enterprise roles.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+from enterprise.constants import ENTERPRISE_ADMIN_ROLE, ENTERPRISE_DATA_API_ACCESS_GROUP, ENTERPRISE_LEARNER_ROLE
+from enterprise.models import EnterpriseCustomerUser, SystemWideEnterpriseRole, SystemWideEnterpriseUserRoleAssignment
+
+
+class Command(BaseCommand):
+    """ management command class """
+    help = 'Assigns existing enterprise users to their specific roles.'
+
+    def handle(self, *args, **options):
+        try:
+            enterprise_admin_role = SystemWideEnterpriseRole.objects.get(name=ENTERPRISE_ADMIN_ROLE)
+            enterprise_learner_role = SystemWideEnterpriseRole.objects.get(name=ENTERPRISE_LEARNER_ROLE)
+        except SystemWideEnterpriseRole.DoesNotExist:
+            raise CommandError('The configured SystemWideEnterpriseRole role does not exist.')
```

Comment:
I don't think we need `try...except` here because `SystemWideEnterpriseRole.objects.get(name=ENTERPRISE_ADMIN_ROLE)` should always return a result. 
Do we expect that the above system wide role will not be present?